runtime/mainnet/src/config/governance.rs

```diff
@@ -80,6 +80,121 @@ impl pallet_sudo::Config for Runtime {
 	type WeightInfo = weights::pallet_sudo::WeightInfo<Runtime>;
 }
 
+/// Contains the impl to set the council members on a runtime upgrade.
+pub(crate) mod initiate_council_migration {
+	use alloc::{vec, vec::Vec};
+
+	use frame_support::traits::OnRuntimeUpgrade;
+	use pallet_collective::WeightInfo;
+	#[cfg(feature = "try-runtime")]
+	use sp_runtime::TryRuntimeError;
+
+	use super::*;
+
+	parameter_types! {
+		C1: AccountId = AccountId::from_ss58check("13BL7T6bTgeEdfEdZqLCKJZPN8ncyFNxxHRKFb2YMATvyfH4").expect("address is valid SS58");
+		C2: AccountId = AccountId::from_ss58check("142zako1kfvrpQ7pJKYR8iGUD58i4wjb78FUsmJ9WcXmkM5z").expect("address is valid SS58");
+		C3: AccountId = AccountId::from_ss58check("14G3CUFnZUBnHZUhahexSZ6AgemaW9zMHBnGccy3df7actf4").expect("address is valid SS58");
+		C4: AccountId = AccountId::from_ss58check("15k9niqckMg338cFBoz9vWFGwnCtwPBquKvqJEfHApijZkDz").expect("address is valid SS58");
+		C5: AccountId = AccountId::from_ss58check("15VPagCVayS6XvT5RogPYop3BJTJzwqR2mCGR1kVn3w58ygg").expect("address is valid SS58");
+		Members: Vec<AccountId> = vec![C1::get(), C2::get(), C3::get(), C4::get(), C5::get()];
+	}
+
+	/// Populates council with certain members.
+	pub struct SetCouncilors;
+
+	impl OnRuntimeUpgrade for SetCouncilors {
+		fn on_runtime_upgrade() -> Weight {
+			let members_in_storage: Vec<AccountId> =
+				pallet_collective::Members::<Runtime, CouncilCollective>::get();
+
+			if !members_in_storage.is_empty() {
+				// Members already set. Nothing to do.
+				return <Runtime as frame_system::Config>::DbWeight::get().reads(1);
+			}
+
+			let mut members: Vec<AccountId> = Members::get();
+			members.sort();
```

Comment:
Given the members are known we can sort out of this code and provide a sorted list.
Less operations to do during the migration.

Reply:
Members are sorted as of [b587c13](https://github.com/r0gue-io/pop-node/pull/499/commits/b587c13d0a5af189b1a540eb3697aa95c7f1f734)